src/graph/xml.cc

```diff
@@ -651,11 +651,9 @@ ncclResult_t ncclTopoGetXmlFromGpu(struct ncclXmlNode* pciNode, uint32_t rocmDev
 
   NCCLCHECK(xmlGetAttrIndex(gpuNode, "sm", &index));
   if (index == -1) {
-    int cudaMajor, cudaMinor;
     cudaDeviceProp devProp;
     CUDACHECK(cudaGetDeviceProperties(&devProp, 0));
-    cudaMajor = devProp.major; cudaMinor = devProp.minor;
-    NCCLCHECK(xmlSetAttrInt(gpuNode, "sm", cudaMajor*10+cudaMinor));
+    NCCLCHECK(xmlSetAttrInt(gpuNode, "sm", devProp.multiProcessorCount));
```

Comment:
Why is this change needed?

Reply:
CU count needs to be put into xml. I didn't want to create new xml just for one field